discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -740,7 +740,10 @@ def prepare_for_installation(self, **kwargs):
             )
 
         self.nodes.notify_iso_ready()
-        self.nodes.start_all()
+        if self._config.is_static_ip and self._config.is_ipv6:
+            self.nodes.start_all_no_check_ips()
+        else:
+            self.nodes.start_all()
```

Comment:
What about changing the `start_all` to `check_ips=False` by default? we should try to minimize the different behavior or different configurations.

Reply:
Didn't want to change default behavior as part of this PR

Reply:
@ori-amizur You don't have to.
Instead of:

```python
    def start_all(self):
        self.run_for_all_nodes("start")

    def start_all_no_check_ips(self):
        self.run_for_all_nodes("start", False)
```
You can:

```python
    def start_all(self, check_ips=True):
        self.run_for_all_nodes("start", check_ips)
```